learn/configuration/instance_options.md

```diff
@@ -32,20 +32,114 @@ The above flag disables analytics for the Meilisearch instance and does not acce
+set MEILI_HTTP_ADDR=127.0.0.1:7700
+./meilisearch
+```
+
+:::
+::::
+
 In the previous example, `./meilisearch` is the command that launches a Meilisearch instance, while `MEILI_DB_PATH` and `MEILI_HTTP_ADDR` are environment variables that modify this instance's behavior.
 
 Environment variables for command-line flags accept `n`, `no`, `f`, `false`, `off`, and `0` as `false`. An absent environment variable will also be considered as `false`. Any other value is considered `true`.
 
 Environment variables are always identical to the corresponding command-line option, but prepended with `MEILI_` and written in all uppercase. **Some options (e.g., `--import-snapshots`) are not available as environment variables.**
 
+## Configuration file
+
+Meilisearch accepts a configuration file in the `.toml` format as an alternative to command-line options and environment variables. Configuration files can be easily shared and versioned, and allow you to define multiple options.
+
+**When used simultaneously, environment variables overwrite the configuration file, and command-line options overwrite environment variables.**
+
+You can download the configuration file using the following command:
+
+```sh
+curl https://raw.githubusercontent.com/meilisearch/meilisearch/main/config.toml > config.toml
+```
+
+By default, Meilisearch will use `config.toml` in the working directory as the default configuration file. You will see it when you launch Meilisearch:
+
+```
+888b     d888          d8b 888 d8b                                            888
+8888b   d8888          Y8P 888 Y8P                                            888
+88888b.d88888              888                                                888
+888Y88888P888  .d88b.  888 888 888 .d8888b   .d88b.   8888b.  888d888 .d8888b 88888b.
+888 Y888P 888 d8P  Y8b 888 888 888 88K      d8P  Y8b     "88b 888P"  d88P"    888 "88b
+888  Y8P  888 88888888 888 888 888 "Y8888b. 88888888 .d888888 888    888      888  888
+888   "   888 Y8b.     888 888 888      X88 Y8b.     888  888 888    Y88b.    888  888
+888       888  "Y8888  888 888 888  88888P'  "Y8888  "Y888888 888     "Y8888P 888  888
+
+Config file path:       "./config.toml"
+```
+
+You can override the default location of the configuration file using the `MEILI_CONFIG_FILE_PATH` environment variable or the `--config-file-path` CLI option:
+
+:::: tabs
+::: tab CLI
+
+```sh
+./meilisearch --config-file-path="./config.toml"
+```
+
+:::
+
+::: tab Environment variable
+
+Linux/MacOS:
+
+```sh
+export MEILI_CONFIG_FILE_PATH="./config.toml"
+./meilisearch
+```
+
+Windows:
+
+```sh
+set MEILI_CONFIG_FILE_PATH="./config.toml"
+./meilisearch
+```
+
+:::
+::::
+
+### Configuration file format
+
+You can configure all environment variables and CLI options using a dedicated option in the configuration file. These options follow the [snake case](https://en.wikipedia.org/wiki/Snake_case) convention. For example, `--import-dump` must be named `import_dump` in the file.
```

Comment:
What do you think about removing the "using a dedicated option" bit?

```suggestion
You can configure all environment variables and CLI options using the configuration file. These options must be written in [snake case](https://en.wikipedia.org/wiki/Snake_case). For example, `--import-dump` must be named `import_dump` in the file.
```

Reply:
We're talking about the file format here 🤔  So wouldn't it be better to talk about "options"? We don't mention them anywhere else.

Reply:
I agree w/ Gui's suggestion, and would go a little further.

I also think that after this paragraph we should provide an example of a full line from a config file. Of course, anyone curious enough can look at the default config file we provide and figure it out from that... but people are lazy! This should make the formatting completely clear.

````suggestion
You can configure any environment variable or CLI option using a configuration file. In configuration files, options must be written in [snake case](https://en.wikipedia.org/wiki/Snake_case). For example, `--import-dump` would be written as `import_dump`.

```toml
import_dump = "./example.dump"
```
````